### PR TITLE
Adding "add" button to Domains sidebar item. (A/B Test)

### PR DESCRIPTION
**Pull request by lucasartoni**

### CODE REVIEW

File: client/my-sites/sidebar/sidebar.jsx [hunk 1]
```diff
@@ -279,12 +282,19 @@ module.exports = React.createClass( {
 			return null;
 		}
 
+		if ( abtest( 'domainsAddButton' ) === 'button' ) {
+			addDomainButton = <a onClick={ this.onNavigate } href={ addDomainLink } className="add-new">{ this.translate( 'Add' ) }</a>;
+		} else {
+			addDomainButton = '';
```
Comment (breezyskies) [inline client/my-sites/sidebar/sidebar.jsx]: You could also define this empty string as the default on line 259 above and just have an if statement here to override.

File: client/my-sites/sidebar/sidebar.jsx [hunk 2]
```diff
@@ -22,6 +22,8 @@ var config = require( 'config' ),
 	AdsUtils = require( 'lib/ads/utils' ),
 	Gridicon = require( 'components/gridicon' );
 
+var abtest = require( 'lib/abtest' ).abtest;
```
Comment (breezyskies) [inline client/my-sites/sidebar/sidebar.jsx]: This should be appended to the statement above after the Gridicon requirement.

File: client/my-sites/sidebar/sidebar.jsx (hunk 1 above)
Comment (lucasartoni) [inline client/my-sites/sidebar/sidebar.jsx]: done

File: client/my-sites/sidebar/sidebar.jsx (hunk 2 above)
Comment (lucasartoni) [inline client/my-sites/sidebar/sidebar.jsx]: done

**Comment by lucasartoni** [comment]
domainsaddbutton_2016013-original
domainsaddbutton_2016013-button

To test the two variations:

``localStorage.setItem('ABTests','{"domainsAddButton_20160113":"button"}')``
``localStorage.setItem('ABTests','{"domainsAddButton_20160113":"original"}')``

**Comment by breezyskies** [comment]
LGTM :+1:

**Pull request merged**